jigglypuffRL/common/policies.py

```diff
@@ -8,28 +8,64 @@
 # TODO (ajaysub110): add other policy classes
 
 
-class MlpPolicy(nn.Module):
+class BasePolicy(nn.Module):
     """
-    Policy object that implements actor critic, using a MLP (2 layers of 24)
+    Base policy that all other policies should inherit.
     :param env: (Gym environment) The environment to learn from
-    :param n_hidden: (int) number of neurons in hidden layers
     """
 
-    def __init__(self, env, n_hidden=16):
-        super(MlpPolicy, self).__init__()
+    def __init__(self, env):
+        super(BasePolicy, self).__init__()
 
-        self.n_hidden = n_hidden
         self.state_space = None
         self.action_space = None
         self.env = env
 
         if isinstance(env.observation_space, gym.spaces.Box):
             self.state_space = env.observation_space.shape[0]
+        else:
+            print("Env type {} is not supported yet".format(type(env)))
 
         if isinstance(env.action_space, gym.spaces.Discrete):
             self.action_space = env.action_space.n
         elif isinstance(env.action_space, gym.spaces.Box):
             self.action_space = env.action_space.shape[0]
+        else:
+            print("Env type {} is not supported yet".format(type(env)))
```

Comment:
Instead of printing, can we raise a `NotImplementedError` if the env type is not supported yet?